requirements.txt

```diff
@@ -17,7 +17,8 @@ langdetect >=1.0.0,<=1.0.9
 dataclasses_json >=0.5.4
 autopep8 >=1.5.7
 dill ==0.3.4
-sktime >=0.5.0,<=0.8.0
+sktime >=0.11.0
```

Comment:
Can you upper cap these to avoid (potential) breaking changes without us necessarily releasing a new version? It has happened in the past :) 

I think this

```python
sktime >=0.11.0,<0.12.0
statsforecast >=0.5.3,<0.6.0
```

should work.

Reply:
Sure, fixed in [be7b9ae](https://github.com/mindsdb/lightwood/pull/861/commits/be7b9ae003f4690d8b3b7572f2d488550c05cffd). :D